content/news/2022_01_30_GRASS_GIS_8_0_0_released.md

```diff
@@ -0,0 +1,179 @@
+Furthermore, semantic labels can also be passed in a text file
+when registering raster maps into time series by means of
+[t.register](https://grass.osgeo.org/grass80/manuals/t.register.html).
+
+There's a new raster module, 
+[r.in.pdal](https://grass.osgeo.org/grass80/manuals/r.in.pdal.html), 
+a PDAL based replacement of r.in.lidar 
+that allows to import any point dimension (including user defined 
+ones e.g. from PDAL filters), supports 19 binning methods (including
+eigenvalues) and point filtering by values in any dimension.
+
+<a href="/images/gallery/remote_sensing/band_references_scheme.png">
+  <img src="/images/gallery/remote_sensing/band_references_scheme.png"
+   alt="New enhanced classification workflow"
+   title="New enhanced classification workflow"
+   width="45%" style="float:right;padding-left:10px;padding-top:10px">
+</a>
+
+#### Imagery modules:
+
+All raster classification modules now generate signature files with
+embedded semantic labels. This allows to apply the signature file from
+one imagery scene (group) to any number of other scenes as long as they
+consist of the same bands (as identified by their semantic labels).
+This makes it so **much easier to classify different imagery groups 
+with identical semantic labels**.
+
+The new module 
+[i.band.library](https://grass.osgeo.org/grass80/manuals/i.band.library.html) 
+stores and prints available semantic labels used for multispectral 
+data. So far only system-defined band references are supported, 
+i.e., Landsat 5, Landsat 7, Landsat 8 and Sentinel 2.
+
+#### Temporal GIS modules
+
+With the addition of semantic labels, the temporal database was modified to
+version 3 and hence, to be able to read and process GRASS 7.x space-time
+datasets, users will be prompted to run the new 
+[t.upgrade](https://grass.osgeo.org/grass80/manuals/t.upgrade.html) module.
+If users want to read space-time datasets newly created in GRASS 8 back
+in GRASS 7.x, they can run
+[t.downgrade](https://grass.osgeo.org/grass78/manuals/t.downgrade.html).
+
+#### General modules
+
+[g.extension](https://grass.osgeo.org/grass80/manuals/g.extension.html),
+the tool allowing to install extensions, received extensive changes
+to adapt to the new [addons repository](https://github.com/OSGeo/grass-addons) 
+structure with branches pointing to different GRASS major versions.
+This change introduced the possibility to **install pull request branches**
+making it so much faster to test new changes.
+
+#### Scripting and code quality
+
+Major efforts have been made to port all Python scripts to Python 3.6.
+Also, there's a new Python subpackage for working with locations and mapsets.
+All Python code has been formatted with `black` and checked against `flake8`.
+Most warnings in C code were removed and are now checked and, all code 
+is tested with CodeQL.
+
+#### PROJ and GDAL support
+
+PROJ 4 to PROJ 7 compatibility has been implemented. WKT2 can be 
```

Comment:
```suggestion
PROJ 4 to PROJ 8 compatibility has been implemented. WKT2 can be 
```

(on Fedora 35 I work with PROJ 8.2.1, January 1st, 2022)

Reply:
> (on Fedora 35 I work with PROJ 8.2.1, January 1st, 2022)

Same for me with MacPorts.